core/src/main/java/org/elasticsearch/common/io/stream/StreamOutput.java

```diff
@@ -872,6 +872,16 @@ public void writeList(List<? extends Writeable> list) throws IOException {
     }
 
     /**
+     * Writes a list of strings
+     */
+    public void writeStringList(List<String> list) throws IOException {
```

Comment:
yea let's do it in a separate change, I was annoyed here that I could read a string list in one line but not write it. I made that work.